sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveExplainSuite.scala

```diff
@@ -182,4 +183,27 @@ class HiveExplainSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       assert(output.toString.contains(s"Scan hive default.$tableName"))
     }
   }
+
+  test("SPARK-26661: Show actual class name of the writing command in CTAS explain") {
+    Seq(true, false).foreach { convertCTAS =>
+
+      withSQLConf(
+          HiveUtils.CONVERT_METASTORE_CTAS.key -> convertCTAS.toString,
+          HiveUtils.CONVERT_METASTORE_PARQUET.key -> convertCTAS.toString) {
+
+        val tableName = "tab1"
```

Comment:
I would do:

```scala
        val keyword = if (convertCTAS) {
          Utils.getSimpleName(classOf[InsertIntoHadoopFsRelationCommand])
        } else {
          Utils.getSimpleName(classOf[InsertIntoHiveTable])
        }
        val df = sql("EXPLAIN CREATE TABLE tab1 STORED AS PARQUET AS SELECT * FROM range(2)")
        checkKeywordsExist(df, keyword)
```

Reply:
or `OptimizedCreateHiveTableAsSelectCommand` and `CreateHiveTableAsSelectCommand`. Checking one should be okay.

Reply:
Thank you very much! This does look cleaner.
But I would still like to explicitly check both the `Execute xxxCommand` part and the `InsertIntoXXX` part, to emphasis that this change is about making the latter more consistent with the former. Is that okay with you?